MSAL/MSAL.xcodeproj/project.pbxproj

```diff
@@ -2895,7 +2901,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.microsoft.InteractiveiOSTests;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SDKROOT = iphoneos;
-				SWIFT_VERSION = 3.0;
+				SWIFT_VERSION = 4.2;
```

Comment:
Eventually, but it will require some swift code updates, which should be a separate PR. Our only swift code is in automation, so I don't see an urgency in this.